src/components/Card.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+class Card extends React.Component {
+
+  render() {
+    return (
+      <div className={ this.props.modifier ? 'p-card--' +  this.props.modifier : 'p-card' }>
+        <header className="p-card__header">
+          <img src={ this.props.imgSrc } alt="placeholder" />
```

Comment:
Probably want the "placeholder" to be a property too.

Might be worth passing image props as an object with `src` and `alt` as keys.